news/2024/2024-05-20-the-followpoint-yesmydarknesss-the-osu-catch-prodigy.md

```diff
@@ -0,0 +1,95 @@
+>
+> Yes, after you move past the spamming phase, you need to find your own reasons for your gameplay. Why did you miss? How do you catch the patterns? What beat should you use?
+> In my case, when I missed a section, I kept trying to change my beat. If I was tapping my keyboard 1/4 at that moment, I would switch to 1/3 or 1/6, constantly adjusting my timing. Once I found the right beat, I memorized it and kept it in mind when retrying that map.
+>
+> After that, I focus on the hard patterns of the map and replay those parts over and over. If a map feels easy to full combo or play except for one difficult pattern or part, then that part is your weakness. In rhythm games, if you're a picky eater, you won't survive.
+>
+> Finally, the way to practice in osu!catch (hurray). In this game mode there is special tech called 'pixel', where you have to catch the end of one fruit and move to the other end of the next fruit. You have to control your catcher really accurately during these patterns.
+> So, sometimes it's better to play easy maps, but catch the fruits with the edge of the catcher. I sometimes practice by catching all patterns with the right end, left end, or both ends of the catcher. This helps you get comfortable with the 'pixel' technique, making your movements smoother and improving your timing in Catch the Beat.
+
+*What is your favorite osu!catch score you’ve set?*
+
+> Hmmm, there are too many scores to decide. My favorite catch score is probably [Chroma - I [AI] +HR](https://osu.ppy.sh/scores/1956565686)? My other scores can be replaced by other pro players, who play better than me, but I still can't understand how did I do that CS5.3 with HR.
+
+*What made you play Xronier map using "[guitar hero playstyle](https://www.twitch.tv/yesmydarknesss/clip/InnocentAbstemiousYakPupper-vX89K6koYLVGQQdP)" that one time?*
+
+> I've played a few times in guitar hero style even before Xronier just for fun, and one of my best scores done in this playstyle on [ADHD](https://osu.ppy.sh/beatmapsets/1789527#fruits/3666753) map is also my personal [Hidden mod pp record](https://osu.ppy.sh/scores/1957545279).
+
+*Who would you call the best osu!catch player(s) at the moment and why?*
+
+> In this generation I would call [Motion](https://osu.ppy.sh/users/3885626) the best player. He has mastered multiple techniques, crazy speed, and even a lot more clean movement. A complete all-rounder.
+
+*What would you call the most impressive skillset in catch?*
+
+> I'm really sad to say it, but it's probably the "pixel" technique, which I often called a "luck issue" before. It's unique to osu!catch, and you must practice this tech, because it will be hard to improve later if you find it difficult.
```

Comment:
```suggestion
> I'm really sad to say it, but it's probably the "pixel dash", which I often called a "luck issue" before. It's unique to osu!catch, and you must practice this tech, because it will be hard to improve later if you find it difficult.
```